### ISSUE TITLE
Backward navigation

### ISSUE THREAD

**Issue by hugueslacroix** [hugueslacroix/tamaudio#2]
Se debería poder volver atrás para arreglar algo o poner una información que faltó. (a mí se me olvidó poner los otros idiomas que hablo, por ejemplo).

**Comment by hugueslacroix** [comment]
This is a pending feature of the jsPsych library. We'll have to wait. 
To make sure they don't press continue without filling the form, I'll add the "required" properties on the mandatory fields.

**Comment by hugueslacroix** [comment]
@Maura04, ¿me podrías escribir aquí todos los campos obligatorios que necesitamos?

**Comment by Maura04** [comment]
In page 1, all of them: 
![image](https://cloud.githubusercontent.com/assets/27859211/25282746/6351a562-267f-11e7-92d6-338712b5044e.png)

**Comment by Maura04** [comment]
De hecho, todos los campos del cuestionario excepto los comentarios de la última página. En algunos se les puede sugerir que pongan N/A

**Comment by hugueslacroix** [comment]
Ok, propongo lo siguiente, todos los campos obligatorios, con algunas precisiones:  
Página 1: 
-"Sexo", agregar una tercera opción, ¿Qué podríamos poner? ¿N/A?
-"Fecha de llegada a Canadá (si naciste en otro país)" (opcional) 
- "¿En cuáles y durante cuánto tiempo?" (opcional y cambiar por un campo más grande (textarea))

Página 2: 
-"¿Desde qué edad?" (opcional)
-Posibilidad de suprimir una pesteña (tab) de lengua si la persona solo habla dos lenguas por ejemplo. ¿Dejamos tres pestañas como "default"?

Pp. 3-4-5 : todo obligatorio

P. 6 
Opcional. 

¿Qué te parece?

**Comment by Maura04** [comment]
Me parece bien. Pero en ""Fecha de llegada a Canadá (si naciste en otro país)" (opcional)", quizás sea mejor obligatorio con la opción N/A, o que sea obligatorio solo si nació en otro país.

**Comment by hugueslacroix** [comment]
No hay posibilidad de poner "obligatorio de manera condicional", entonces los que dependen de una respuesta previa son todos opcionales.

**Comment by Maura04** [comment]
Sí, pero ese dato es importante para clasificar al participante en heritage o nativo, y en secuencial o simultáneo. Entonces, si no escribe ese dato, sería bastante grave. Es verdad que el dato de la edad de adquisición del francés es útil también para clasificarlos, pero no es lo mismo aprender francés en una escuela de su país de origen que en Montreal. Me preocupa un poquito eso. Lo otro que podemos hacer es contactar al participante (por email) si olvida poner algún dato importante.

**Comment by hugueslacroix** [comment]
@Maura04, puedes consultar la última versión en http://tamaudio.transling.ca/, ¿Qué te parece?
